packages/sysutils/busybox/scripts/getedid

```diff
@@ -0,0 +1,274 @@
+#!/bin/sh
+#
+#
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016 Neil MacLeod (milhouse@kodi.tv)
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
++################################################################################
+#
+#
+# exit 1 = unsupported GPU
+# exit 2 = dual boot system
+# exit 3 = no backup file, therefore no restore
+# exit 4 = extlinux.conf and syslinux.cfg are available
+# exit 5 = changes are already made either to extlinux.conf or syslinux.cfg
+# exit 6 = xorg.conf already exists in /storage/.config
+# exit 7 = no xorg.conf in /storage/.config
+
+
+# Help message and usage explanation
+help() {
+echo "This script generates a custom EDID depending on your GPU"
+echo ""
+echo "To check which GPU you are using, use: ./getedid gpu"
+echo ""
+echo "To create a custon EDID, just use this script like: ./getedid start"
```

Comment:
`getedid start` is a little odd, what am I starting? Why not `getedid create` as this more in keeping with the other `delete` and `restore` options.